packages/wdio-utils/src/envDetector.ts

```diff
@@ -99,6 +100,9 @@ function isMobile(capabilities: WebdriverIO.Capabilities) {
         capabilities.platformName && capabilities.platformName.match(/ios/i) ||
         capabilities.platformName && capabilities.platformName.match(/tvos/i) ||
         capabilities.platformName && capabilities.platformName.match(/android/i) ||
+        bsOptions.platformName && bsOptions.platformName.match(/ios/i) ||
```

Comment:
I know you extended the current logic but instead of returning a match, we want to test if there is a match which is cheaper in performance as well so let's change all of these like so:
```suggestion
        /ios/i.test(bsOptions.platformName) ||
```

Reply:
Ok, thank you, but why we use this command instead the other? 
I read doc of the test() function, but I didn't get it.
We don't need to check if platformName exist?